### ISSUE TITLE
Remove duplicate code in SimplePager

### ISSUE THREAD

**Issue by pulzarraider** [sonata-project/SonataDoctrinePhpcrAdminBundle#320]
In PR sonata-project/SonataAdminBundle#2656 I have copied the SimplePager from this bundle to SonataAdminBundle as it is very nice feature that should be available also for other users. SimplePager will be available in SonataAdmin 2.4.

To have less code duplicated after the SonataAdmin 2.4 will be released I suggest to remove duplicate code in SonataDoctrinePhpcrAdminBundle's SimplePager and extend the class from SonataAdminBundle's SimplePager. This will be a BC change.

``` php
namespace Sonata\DoctrinePHPCRAdminBundle\Datagrid;

use Sonata\AdminBundle\Datagrid\SimplePager as BaseSimplePager;

class SimplePager extends BaseSimplePager
{
...
}
```

Maybe we can also mark the SonataDoctrinePhpcrAdminBundle's SimplePager as deprecated if the code of SimplePager will be same after the SonataAdmin's release.

**Comment by ElectricMaxxx** [comment]
Solved through #416

**Comment by ElectricMaxxx** [comment]
Solved through #416
